src/firmware/boards/legacy_robot_stm32f4/main.c

```diff
@@ -59,6 +59,8 @@
 #include "io/usb_config.h"
 #include "io/wheels.h"
 #include "priority.h"
+#include "shared/2015_robot_constants.h"
+#include "shared/2015_wheel_constants.h"
```

Comment:
nit; duplicate include

Reply:
robot_constants vs wheel_constants

Reply:
I just went ahead and merged the two files